packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -165,6 +166,14 @@ export interface BucketDeploymentProps {
    */
   readonly serverSideEncryptionCustomerAlgorithm?: string;
 
+  /**
+   * Sets the ACL for the object when the command is performed.
+   * If you use this parameter you must have the "s3:PutObjectAcl" permission included in the list of actions for your IAM policy.
```

Comment:
Which IAM policy should have this permission configured?

I'm not very familiar with this module but usually the CDK ensures that the permissions are correctly configured.

Reply:
You are right. This permission is granted by construct. I have removed this sentence.